src/clandinin_lab_to_nwb/brezovec/brezovecimaginginterface.py

```diff
@@ -0,0 +1,90 @@
+from dateutil.parser import parse
+from clandinin_lab_to_nwb.brezovec.brezovecimagingextractor import BrezovecMultiPlaneImagingExtractor
+
+from roiextractors import MultiImagingExtractor
+from neuroconv.datainterfaces.ophys.baseimagingextractorinterface import BaseImagingExtractorInterface
+from neuroconv.utils import FolderPathType
+from neuroconv.utils.dict import DeepDict
+
+
+class BrezovecMultiPlaneImagingInterface(BaseImagingExtractorInterface):
+    """
+    Data Interface for writing the Clandinin lab imaging data to NWB file using BrezovecMultiPlaneImagingExtractor.
+    """
+
+    Extractor = BrezovecMultiPlaneImagingExtractor
+
+    def __init__(self, folder_path: FolderPathType, stream_name: str, verbose: bool = True):
+
+        """
+        Initialize reading of NIfTI files.
+
+        Parameters
+        ----------
+        folder_path : FolderPathType
+            The path to the folder that contains the NIfTI image files (.nii) and configuration files (.xml) from Bruker system.
+        stream_name: str, optional
+            The name of the recording channel.
+        verbose : bool, default: True
+        """
+        super().__init__(
+            folder_path=folder_path,
+            stream_name=stream_name,
+            verbose=verbose,
+        )
+
+    @classmethod
+    def get_streams(cls, folder_path) -> dict:
+        streams = BrezovecMultiPlaneImagingExtractor.get_streams(folder_path=folder_path)
+        return streams
+
+    def get_metadata(self) -> DeepDict:
+        metadata = super().get_metadata()
+
+        xml_metadata = self.imaging_extractor.xml_metadata
+        session_start_time = parse(xml_metadata["date"])
+        metadata["NWBFile"].update(session_start_time=session_start_time)
+
+        description = f"Version {xml_metadata['version']}"
+        device_name = "BrukerFluorescenceMicroscope"  # TODO doucle check in the paper
+        metadata["Ophys"]["Device"][0].update(
+            name=device_name,
+            description=description,
+        )
+
+        imaging_plane_metadata = metadata["Ophys"]["ImagingPlane"][0]
+        imaging_plane_metadata.update(
+            device=device_name,
+            imaging_rate=self.imaging_extractor.get_sampling_frequency(),
+        )
+        two_photon_series_metadata = metadata["Ophys"]["TwoPhotonSeries"][0]
+        two_photon_series_metadata.update(
+            description="Imaging data acquired from the Bruker Two-Photon Microscope and transform to NIfTI.",  # TODO doucle check in the paper
+            unit="px",
+            format=".nii",
```

Comment:
```suggestion

```
If I remember correctly, specifying `format` is only useful if you'd be using `external_file` mode, and it's not needed here (we removed from Bruker too). @CodyCBakerPhD

Reply:
Ok, thanks! I removed the format spec in the other branch (add_imaging_interfaces)